### PR TITLE
Logainenu to Development merge

### PR DESCRIPTION
**Pull request by angryoctopode**

### CODE REVIEW

**Review by eristow — APPROVED** [review]
Looks all good to me.

**Review by wcampbell09 — APPROVED** [review]
Code looks good! 👍

File: modules/core/client/views/mainmenuadmin.client.view.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            </p>
+            <p>
+              <a class="btn btn-primary btn-lg" ui-sref="somethingelse">Inventory Management</a>
+            </p>
+            <p>
+              <a class="btn btn-primary btn-lg" ui-sref="admin.users">User Management</a>
+            </p>
+          </div>
+        </div>
+      </section>
```
Comment (dkwhite200) [inline modules/core/client/views/mainmenuadmin.client.view.html]: Fix this -_-

**Review by dkwhite200 — APPROVED** [review]
We gucci

**Review by AriTheGuitarMan — APPROVED** [review]
Good job with routes, schema, and title bar. Make sure controller is all good when tested.

**Pull request merged**